### PR TITLE
Issue#12: Use `keyring-backend=file` by default to create keys.

### LINKED ISSUE #12: Keys are not created on Linux
On OSX, Cosmos SDK uses `keychain` to store keys by default.
So when `archwayd` try to create key, OS automatically ask use for password by OS level.
But on Linux, it uses `file` to store keys and that means `archwayd` should handle prompt to receive password from user.

To make scenario simple, use `file` keyring-backend by default for all OS.

### PR DESCRIPTION
**Pull request by rootwarp**
Add `keyring-backend=file` option to store keys into file.

close #12

### CODE REVIEW

File: src/networks/data/local.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "network": {
     "name": "local",
-    "chainId": "localtest",
+    "chainId": "localnet-1",
```
Comment (Choi-Jinhong) [inline src/networks/data/local.json:4]: I think this need to othter commit .. don't you @rootwarp?

Comment (rootwarp) [reply src/networks/data/local.json:4]: @Choi-Jinhong Do you prefer divide this into diff. issue?

Comment (Choi-Jinhong) [reply src/networks/data/local.json:4]: Nope. I knew we have to divide diff issue. But I don't care that this isn't mandatory :D

Comment (rootwarp) [reply src/networks/data/local.json]: I think it could be better apply this on another issue as your opinion.

Comment (rootwarp) [reply src/networks/data/local.json]: @Choi-Jinhong  I reverted default chainId. Please check again.

**Review by Choi-Jinhong — APPROVED** [review]
LGTM !

**Pull request merged**